### DISCUSSION TITLE
Help with saving to category folders, please

### DISCUSSION THREAD

**Discussion by Neobond** [nzbgetcom/nzbget#443] [Q&A] [answered]
I can't seem to get categories working. Downloads for series and movies save to the completed folder, but not the sub folder movies or series as defined below 

![2024-11-26_145816](https://github.com/user-attachments/assets/682f8981-7d8b-42c3-b6e5-20e727ab98a1)

I want to add 'movies' category to the NZBGet download client in Radarr so that it does not monitor downloads in the root 'completed' folder.

Should I be specifying the full path in the category DestDir, like so: ${MainDir}/completed/movies ?

and then this in Radarr (category)?
![2024-11-26_150250](https://github.com/user-attachments/assets/a4d6b347-9808-4860-a607-60a2b75200ac)

Wonder if anyone can assist me in what I am doing wrong?

**Comment by fryfrog** [comment]
I think you want to leave it blank and *enable* the `AppendCategoryDir` option where ever it is.

**Comment by Neobond** [comment]
That is exactly what I did, but movies save to DestDir not the sub 'movie' folder.

I added movie myself for the screenshot, because that's where I want completed movies to download to (so Radarr only ever monitors that folder).

Edit: AppendCategoryDir is enabled.

**Comment by liv-mrr** [comment, marked as answer]
Hi, Neobond
You need to configure categories in NZBGet, provide a full path to them (answering your question) ex: 
![Untitled](https://github.com/user-attachments/assets/3bbadabe-fed7-447e-bcd2-d862f12343b5)
The names of your categories in Radarr and NZBGet must match
Also don't forget to set up root folder in Radarr (It's the folder where Radarr will place your movies and monitor them)

There is another way to sort files. You can use Videosort Extension in NZBGet it's very simple to configure

**Comment by Neobond** [comment]
Thanks I appreciate it. I spent hours messing with the arrs docker project root folder and linking download client and eventually got it working, so I did not want to mess with them again until I know for sure what is what. I even think I can remove the remote path mappings in Download Client settings page in radarr?

![2024-11-27_154100](https://github.com/user-attachments/assets/847df5d3-a9dc-4f97-a283-f9514fac0583)

Radarr's media root folder is this (even though it can monitor folders above it)
![2024-11-27_154340](https://github.com/user-attachments/assets/64c8c54e-5f68-4b92-8671-8467ba10d033)

The Docker containers root is top level media/download client folder:
![2024-11-27_154356](https://github.com/user-attachments/assets/6f1e0d1e-e8c1-4a87-a9b2-9e8dced470b4)

in .yaml given as /volume2/data:/data (which also applies to Sonarr, , nzbget, bazarr, unpackerr)

in qbittorrent and nzbget the root folder in .yaml is given as /volume2/data/usenet:/data/usenet and /volume2/data/torrents:/data/torrents

Maybe I should also be setting the download clients to /data as well and defining the sub folder in the download client itself so that all the linked containers start at /data?

**Comment by Neobond** [comment]
It worked, thanks! 😁
